yarn-project/end-to-end/src/guides/up_quick_start.sh

```diff
@@ -16,36 +16,39 @@ aztec-cli deploy \
 # docs:end:deploy
 
 aztec-cli check-deploy --contract-address 0x03b030d48607ba8a0562f0f1f82be26c3f091e45e10f74c2d8cebb80d526a69f
+# docs:start:declare-contract
```

Comment:
This tag is created but i dont see where it is used?

Reply:
This should actually be part of this PR: https://github.com/AztecProtocol/aztec-packages/pull/2322